diktat-rules/src/test/resources/test/paragraph2/kdoc/ConstructorCommentKDocExpected.kt

```diff
@@ -187,14 +169,11 @@ class A constructor(
  * kdoc
  * class
  * comment
- *
  * @param K
  * @property openName single-line comment
- * @property openLastName
- *   block
```

Comment:
I think it was moved to a newline with a some purpose? Isn't it a newline rule that split long lines?

Reply:
This was an early implementation, as the final implementation for a multiline property description, it was decided to start it from the same line, rather than from a new one.